packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -212,7 +212,8 @@ describe('plugin-meetings', () => {
       };
 
       const progressTime = async () => {
-        await clock.tickAsync(initialConfig.analyzerInterval);
+        // needed after analyzerInterval to execute the task in the queue
+        await clock.tickAsync(initialConfig.analyzerInterval * 1.5); // extra time
```

Comment:
I'm running the tests with this `* 1.5` and they still pass. Are you sure this is necessary?

Reply:
I fixed the fallback timeOut, this modification not needed anymore